src/components/Layout/ConstrainedWidth.js

```diff
@@ -14,6 +14,7 @@ const ConstrainedWidth = ({ children }) => (
         margin-right: auto;
         border-radius: 0 0 38px 0;
         padding-bottom: 2rem;
+        ${paddingTop ? `padding-top: ${paddingTop};` : ''}
```

Comment:
Is this border radius in use here? I don't think I would expect this component to provide any vertical padding, it's been extracted as a utility to control the page width.

Reply:
I think I also used this component as the Project cards which have one curved edge. I could create a separate component for the project cards ?